layers/+readers/deft/packages.el

```diff
@@ -3,14 +3,54 @@
 ;; Copyright (c) 2012-2018 Sylvain Benner & Contributors
 ;;
 ;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;;         Bruno Morais <brunosmmm@gmail.com>
 ;; URL: https://github.com/syl20bnr/spacemacs
 ;;
 ;; This file is not part of GNU Emacs.
 ;;
 ;;; License: GPLv3
 
 (setq deft-packages
-  '(deft))
+      '(
+        deft
+        zetteldeft
+        ))
+
+(defun deft/init-zetteldeft ()
+  (use-package zetteldeft
+    :if (eq deft-zetteldeft t)
```

Comment:
Please supress installing the package by adding a toggle directive in the `deft-packages` list rather than just not loading it if it is not used.